packages/zapp/console/src/components/flytegraph/ReactFlow/customNodeComponents.tsx

```diff
@@ -349,12 +349,18 @@ export const ReactFlowCustomTaskNode = ({ data }: any) => {
           if (!logsByPhase.has(phase)) {
             return null;
           }
-
+          const defaultColor = getStatusColor();
+          const phaseColor = getStatusColor(phase);
+          const color = selectedPhase
+            ? phase === selectedPhase
+              ? phaseColor
+              : defaultColor
+            : phaseColor;
```

Comment:
nit: it's hard to read ternary operators when there is more than one of them in a statement. suggested fixes:

- `const color = !selectedPhase || selectedPhase === phase ? phaseColor : defaultColor;`
- Or using if/else statement
- Or create a separate function to return color with if checks and early returns.

Reply:
updated ternary!